Lombiq.OrchardCoreApiClient.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -89,10 +90,17 @@ public static class TestCaseUITestContextExtensions
         ApiClient apiClient,
         TenantApiModel createApiModel)
     {
-        await apiClient.OrchardCoreApi.CreateAsync(createApiModel);
-
-        await context.GoToAdminRelativeUrlAsync("/Tenants");
-        context.Exists(By.LinkText(createApiModel.Name));
+        using (var response = await apiClient.OrchardCoreApi.CreateAsync(createApiModel))
+        {
+            response.Error.ShouldBeNull(
+                $"Tenant creation failed with status code {response.StatusCode}. Content: {response.Error?.Content}\n" +
+                $"Request: {response.RequestMessage}\nDriver URL: {context.Driver.Url}");
+
+            // Check if response URL is valid, and visit it (should be the tenant setup page and not 404 error).
+            var responseUrl = new Uri(response.Content);
+            responseUrl.AbsolutePath.ShouldBe($"/{createApiModel.Name}", StringCompareShould.IgnoreCase);
+            await context.GoToAbsoluteUrlAsync(responseUrl);
+        }
```

Comment:
How does this fix the test? It has more assertions then previously, but why wouldn't it fail in the same way?

Reply:
It doesn't, this is just more diagnostics, since it doesn't fail for me locally. I can't tell what's wrong just yet, because the version in NuGet is also bugged so the NuGet tests fail before we reach this point in the main run. :(

Reply:
I see, I just understood this as being done due to the status of the issue.

Reply:
You can BTW just remove the NuGet build temporarily from the GHA workflow.